frontend/src/app/services/label.service.ts

```diff
@@ -7,7 +7,7 @@ import {Point} from '../model/Point';
 import {Label} from '../model/labels/Label';
 import {LabelTag} from '../model/labels/LabelTag';
 import {SliceSelection} from '../model/selections/SliceSelection';
-import {ROISelection2D} from '../model/selections/ROISelection2D';
+import {RectangleSelection} from '../model/selections/RectangleSelection';
```

Comment:
Hi @Darkoon let me know if `RectangleSelection` is a good choice or do we want to bring the `2D` back. I'm still not sure 🐱

Reply:
It's ok 🥇